### ISSUE TITLE
Paging -help output

### ISSUE THREAD

**Issue by thrust26** [stella-emu/stella#962]
With the ever increasing number of features and resulting command line parameters, the output for `-help` has become quite large. Therefore we should page the help text, where the output stops after each page.

I tried to do this for windows, but the way Windows attaches a console to a GUI app is **crazy**. 😒 Also MSVS behaves different between Debug and Release mode, so I cannot even debug the problems. 😠 Whatever I tried, it always resulted into having to press `Enter` twice **and** original console command prompt cluttering the output:
![image](https://user-images.githubusercontent.com/24360774/226111372-ba9ed1eb-4370-4d8d-aa03-4987ab86cdb9.png)

@sa666666 
Maybe this can be easier added at least for Linux and MacOS?

**Comment by DirtyHairy** [comment]
Hm, that would be highly unusual behaviour on UNIX. Many, many programs have help output that exceeds a typical screen, and the usual thing to do is to just pipe the output through `less` (or `more`).

**Comment by thrust26** [comment]
Yes, Windows is special here. A GUI doesn't have any console output by default. So you have to attach a separate console. And then you can redirect `stdout` and `stderr` to that console. But for whatever reason, `stdin` doesn't work. From searching the web I found that Microsoft has changed (or broken?) this.

**Comment by sa666666** [comment]
I was going to say the same as @DirtyHairy.  This is more properly handled by the shell/console from which Stella is launched, not by the app itself.

**Comment by thrust26** [comment]
Not with Windows and MSVS. Without attaching a separate console, there is **NO** console output at all in a GUI application like Stella.

**Comment by thrust26** [comment]
@sa666666 @DirtyHairy 
I just committed code which should work with Linux and MacOS. It compiles but can you give it a try please?

**Comment by sa666666** [comment]
Hmm, I left a comment, and now it's gone.  I originally said that this is working fine in Linux (KDE/konsole shell).  And bonus points that it still works with `less` and `more`.

**Comment by thrust26** [comment]
Thanks, I was worried that I might have broken something. 

For Windows the really working solution seems to require to create two programs. A prime example is MSVS, it has `devenv.com` and `devenv.exe`. From the command line, using `devenv` defaults to the `.com` version, which is a console app. If required `devenv.com` calls `devenv.exe`. 

Another alternative would be use a GUI window for the console output. `devenv.exe -?` does that.

But I think both solutions might be overkill. @sa666666 Thoughts?

BTW: Meanwhile I added something similar for the debugger's `PromptWideget` (dc780d8e).

**Comment by sa666666** [comment]
> For Windows the really working solution seems to require to create two programs. A prime example is MSVS, it has `devenv.com` and `devenv.exe`. From the command line, using `devenv` defaults to the `.com` version, which is a console app. If required `devenv.com` calls `devenv.exe`.

This is exactly what I do with the HarmonyCart app.  Two versions are included; one is GUI-based, the other is used on the commandline (automatically, as you describe).  I never did find another way to do it.  If you can, that would be great.

I don't know if it's worth the effort for Stella, though.  For HarmonyCart, there's specific functionality at the commandline.  In Stella, it's just showing the help screen.

**Comment by thrust26** [comment]
> In Stella, it's just showing the help screen.

Plus `listrominfo` and `rominfo`. 😄 

If we create such a 2nd app, how would one setup MSVS? Can both be combined into one solution?

**Comment by sa666666** [comment]
Not that I know of.  I never did find a solution beyond just compiling two versions of the app, and naming one with `.exe` and the other with `.com`.

**Comment by thrust26** [comment]
I think its not worth the extra effort. At least not now.
